bitnami/contour/Chart.yaml

```diff
@@ -25,4 +25,4 @@ maintainers:
 name: contour
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/contour
-version: 12.2.3
```

Comment:
It seems that the latest version is _12.2.4_ and not _12.2.3_. Can you update the PR to use the most recent changes of the main branch, please @clayton-gonsalves?